centos/SPECS/softethervpn.spec

```diff
@@ -87,6 +87,25 @@ if [ $1 -eq 0 ]; then
 fi
 
 %changelog
+* Wed Dec 18 2017 You Xiaojie <yxj790222@163.com> - 4.23.9647-1
+- Upgraded OpenSSL to 1.0.2l.
```

Comment:
Question: How did you find this list?

Reply:
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/ChangeLog
and which  version I would using? the version shown by  running binary or this file?